.eslintrc

```diff
@@ -1,15 +1,20 @@
 {
-  "extends": "amo",
+  "extends": [
+    "amo",
+    "prettier",
+  ],
   "globals": {
     "CLIENT_CONFIG": true,
     "webpackIsomorphicTools": true,
     "ga": true,
   },
   "parser": "babel-eslint",
   "plugins": [
-    "jest"
+    "jest",
+    "prettier",
   ],
   "rules": {
+    "prettier/prettier": ["error", {"singleQuote": true, "trailingComma": 'es5'}],
```

Comment:
This duplicate of the .prettierrc conf will eventually be eliminated, Prettier's config usage is new and the eslint plugin needs to catch up.

Reply:
Don't you have to specify `--parser flow`? I cannot remember why I have that in my Prettier settings..